src/git.ts

```diff
@@ -163,4 +163,14 @@ export class Git {
       return res.stdout.trim();
     });
   }
+
+  public static async commitDate(ref: string, gitDir: string | undefined = undefined): Promise<Date> {
+    const args = ['show', '-s', '--format="%ci"', ref];
+    if (gitDir !== undefined) {
+      args.push('--');
+      args.push(gitDir);
+    }
+
+    return new Date(await Git.exec(args));
+  }
```

Comment:
Don't think we need a `gitDir` opt like other methods here:

```ts
  public static async commitDate(ref: string): Promise<Date> {
    return new Date(await Git.exec(['show', '-s', '--format="%ci"', ref]));
  }
```

Reply:
done